test/spec/app_spec.rb

```diff
@@ -767,13 +768,26 @@
     #   skip("Needs to be implemented")
     # end
 
-    # it "should delete tag from tags list (for media and source)" do
-    #   skip("Needs to be implemented")
-    # end
-
-    # it "should edit team" do
-    #   skip("Needs to be implemented")
-    # end
+    it "should delete tag from tags list (for media)", bin1:true  do
+      page = api_create_team_project_and_claim_and_redirect_to_media_page
+      el = wait_for_selector('.media-detail')
+      el.click
+      el = wait_for_selector('.media-actions')
+      el.click
+      el = wait_for_selector('.media-actions__edit')
+      el.click
+      wait_for_selector('ReactTags__selected', :class)
+      fill_field('.ReactTags__tagInput input', "TAG")
+      @driver.action.send_keys("\n").perform      
+      sleep 1
+      fill_field('.ReactTags__tagInput input', "TAG2")
+      @driver.action.send_keys("\n").perform
+      expect(page.tags.length == 2).to be(true)
```

Comment:
Please make sure that the number of tags is 0 before the first tag is added.

Reply:
Ok. Agree